site.yml.sample

```diff
@@ -1,52 +1,88 @@
 ---
 # Defines deployment design and assigns role to server groups
 
+- hosts: all 
```

Comment:
`all` can potentially be harmful if we have other nodes being managed by ansible (other than ceph nodes), so we should explicitly list all the ceph roles.